test/integration/roles/test_amazon_rds/tasks/main.yml

```diff
@@ -0,0 +1,155 @@
+# create and kill an RDS
+
+# Improvements expected in later versions of this test case
+
+# use present instead of create
+# use absent instead of delete
+# use SSD
+
+
+# Start with creation - N.B. if the RDS already exists from a previous run this
+# may not change anything so we don't insist on that
+
+  - name: create RDS
+    tags: test_amazon_rds
+    local_action:
+      module: rds
+      region: "{{ ec2_region }}"
+      command: create
+      instance_name: "{{ resource_prefix }}-rds"
+      db_engine: postgres
+      size: 10
+      instance_type: "db.t2.micro"
+      username: "{{ rds_master_db_user }}"
+      password: "{{ rds_db_master_password }}"
+      publicly_accessible: no
+      wait: yes
+      wait_timeout: 1200 # 600 doesn't cut it!!
+    environment:
+      EC2_REGION: '{{ec2_region}}'
```

Comment:
http://boto3.readthedocs.io/en/latest/guide/configuration.html

I would just allow a `-e profile=xyz` if you wish to override and pass `profile: {{ profile|default(omit) }}`

It's perfectly acceptable to allow `region` to be passed to override default .aws/config region setting (I'd probably name it rds_region rather than ec2_region though)